extension-migrate.py

```diff
@@ -93,21 +107,38 @@ def run_command(cmd):
     cmd += f" --path {old_files[count]}"
 cmd += " --refs refs/heads/filter-source --force"
 
-if args.dry_run:
+if args.debug:
     print(cmd)
+
+if args.dry_run:
+    print("Would have run git filter-repo to filter files")
 else:
-    run_command(cmd)
+    ret = run_command(cmd)
+    if ret.returncode != 0:
+        print(f"Failure: {ret.stderr}")
+        sys.exit(ret.returncode)
```

Comment:
**suggestion (non-blocking):** this block is repeated a bunch. it might be good to move the logic up into `run_command` at some point in the future.